src/main/python/TravelTimeReporter.py

```diff
@@ -411,6 +438,29 @@ def get_drive_alone_time(self):
         dest_terminal_time = self.field2matrix("terminal_time", origin = False)
         self.skims["drive_alone_time"] = orig_terminal_time + self.expand_skim(self.skims["SOV_NT_L_TIME__" + self.settings["time_period"]]) + dest_terminal_time
 
+    def add_active_taz_time(self, bike = True):
+        """
+        Adds the skim values for OD pairs not in the MAZ skim files and reads in the TAZ-level skim values.
+
+        Parameters
+        ----------
+        bike (bool):
+            If set to `True`, obtain the bike skim values. Otherwise obtain the walk skim values.
+        """
+        if bike:
+            mode = "bike"
+            skim = "taz_bike_time"
+        else:
+            mode = "walk"
+            skim = "taz_walk_time"
+
+        self.results["taz_time"] = self.unpivot_skim(skim)
+        self.results[mode] = np.where(
+            self.results[mode] == self.settings["infinity"],
+            self.results["taz_time"],
+            self.results[mode]
+        )
```

Comment:
May be good to remove "taz_time" from self.results after using it to reduce final file size

Reply:
Thanks, I meant to do that but forgot. There may be a few more commits coming as it's crashing during testing.